### DISCUSSION TITLE
ERROR: The local file system returned an error with the following message

### DISCUSSION THREAD

**Discussion by addohm** [abraunegg/onedrive#2619] [General]
What is this error? 

<img width="474" alt="image" src="https://github.com/abraunegg/onedrive/assets/25333107/30f032b9-097d-4d9c-a9bb-c8990782c1d3">

**Comment by abraunegg** [comment]
@addohm 
This error means:

* You are running Docker
* You have created the file system `/onedrive/conf` as one user and the permissions are for that user
* You are running the Docker container as a totally different user with different permissions
* You cannot read|write `/onedrive/conf/refresh_token` thus this is why this error presents itself as the permissions on this file do not allow the user you are running the container as to read|write that file

**Reply by addohm** [reply]
Interesting.  Wonder how that happened.  Theres only one user outside of root on the system and nothing is run as root.  I musta messed up some permissions somewhere.

edit: confirmed.  Somehow some files got root ownership.  That's annoying.  Wonder how that happened.
